structset.go

```diff
@@ -27,7 +27,9 @@ func (s Structset) Apply(doc *Document, mut *Mutation) {
 	for _, field := range s.doc.Fields() {
 		switch field {
 		case pField:
-			continue
+			if doc.IsZeroValue(field) {
```

Comment:
I think you should be able to use `isZero` method that also used in line 65? but yes in this context it looks like not as efficient, since this only check for zero value.
maybe if you can somehow move the check on applyValue method it should work?

this `applyValue` will be more likely to be refactored later when timestamp is moved to separate mutator, and will looks like `applyValue` instead

Reply:
Didn't notice `isZero`. I think it's good enough, then I don't need polyfill too. Let me rework this part.